std/bitmanip.d

```diff
@@ -300,7 +300,7 @@ The example above creates a tagged reference to an Object in the struct A. This
 as $(D taggedPointer), except the first argument which must be a class type instead of a pointer type.
 
 $(RED Warning: Don't use $(D taggedClassRef) with references to garbage collected objects, as it will result in
-undefined behaviour. See $(LINK http://dlang.org/garbage.html) for details.)
+undefined behaviour. See $(LINK ../spec/garbage.html) for details.)
```

Comment:
I just wasted another hour trying to get the DDox pages to build. The furthest I got was

    core.exception.RangeError@../../../../.dub/packages/ddox-0.11.5/source/ddox/parsers/jsonparser.d(458): Range violation

I'm giving up once again. Someone else has to take this on.